ambry-commons/src/main/java/com.github.ambry.commons/ServerErrorCode.java

```diff
@@ -33,5 +33,7 @@
   Unknown_Error,
   Temporarily_Disabled,
   Bad_Request,
-  Retry_After_Backoff
+  Retry_After_Backoff,
+  Ttl_Already_Updated,
```

Comment:
Add `Blob_` as prefix? And use  `Ttl_Already_Updated` in `StoreErrorCode`.

Reply:
Done